issue2pr.py

```diff
@@ -9,10 +9,27 @@
 
 from __future__ import print_function
 
+import argparse
+import getpass
+import json
```

Comment:
Huh, I assumed that you could just pass the dictionary directly in the post request using ``params=``? But maybe not with the GitHub API.